odata-json-format/16 Bound Function.md

```diff
@@ -234,6 +234,27 @@ properties, or just the [entity reference](#EntityReference), as
 appropriate to the action.
 Stream typed parameter values are represented following the same rules as inlined [stream properties](#StreamProperty).
 
+Alternatively, values of non-binding parameters MAY be specified as common expressions
+[OData-URL, section 5.1.1](#ODataURL) that the service evaluates relative to the
+action. They are then encoded as a name/value
```

Comment:
Relative to what are expressions in function invocations evaluated?
```
/service/Employees(23)/CheckAbsence(Employee=@colleague,Date=@date)
?@colleague=Manager
&@date=now() add duration'P1D'
```

Reply:
I'd say the base URL in this example is scheme plus host (and port) plus `/service/Employees(23)/` - note the trailing slash.

Reply:
Seems to be covered now: anything I missed?